nautobot/extras/jobs.py

```diff
@@ -757,20 +760,20 @@ def prepare_job_kwargs(cls, job_kwargs):
         return {k: v for k, v in job_kwargs.items() if k in job_vars}
 
     @staticmethod
-    def load_file(pk):
+    def _load_file_from_proxy(pk):
```

Comment:
*technically* a breaking change to rename these methods but AFAICT no one outside of core is using them, nor should they - these should have been marked as private methods from the start.